docs/commands.md

```diff
@@ -0,0 +1,185 @@
+# Commands List
+
+The following is a list of commands, their arguments, and what they do.
+
+## Administration Commands
+These are commands for the maintenance of LilyBot. They can only be run by server Admins.
+
+### Set Config
+Name: `config set`
+
+Arguments:
+* `guildId` - ID of the guild to set a config for - Guild ID
```

Comment:
There is no longer a `guildId` argument. We now take that from the command context, instead of requiring manual input